frontend/src/pages/groups/allGroupsQuery.graphql

```diff
@@ -1,5 +1,5 @@
-query AllGroups {
-  listGroups {
+query AllGroups($captainId: Int, $groupType: [GroupType!]) {
```

Comment:
Markus built the functionality in the resolver, so this is all it takes to add filtering on the client.